### ISSUE TITLE
Issues with BWM sessions flagged by Matt

### ISSUE THREAD

**Issue by juhuntenburg** [int-brain-lab/paper-brain-wide-map#62]
I'm looking at eid 'c7bd79c9-c47e-4ea5-aea3-74dda991b48e', which has a large number of trials (>1000). 114 of these are error trials, and for every single error trial both firstMovement_times and feedback_times are NaN. The trial intervals aren't unusual (0-10 seconds) so these aren't trials that are timing out. Any idea where the NaNs are coming from?

'a92c4b1d-46bd-457e-a1f4-414265f0e2d4' is another problematic eid - reaction time is negative for every single trial (and therefore this session gets kicked out of every analysis)

**Comment by juhuntenburg** [comment]
'a92c4b1d-46bd-457e-a1f4-414265f0e2d4' is another problematic eid - reaction time is negative for every single trial (and therefore this session gets kicked out of every analysis)

**Comment by GaelleChapuis** [comment]
https://github.com/int-brain-lab/ibllib/tree/develop/ibllib/qc/task_qc_viewer

**Comment by GaelleChapuis** [comment]
This is from session EID :  c7bd79c9-c47e-4ea5-aea3-74dda991b48e

```
2024-02-13 13:58:08 INFO     bpod_trials.py:62   Extracting /Users/gaelle/Downloads/ONE/alyx.internationalbrainlab.org/zadorlab/Subjects/CSH_ZAD_029/2020-09-19/001 as ephys
2024-02-13 13:58:08 INFO     bpod_trials.py:65   ephys session on _iblrig_zadorlab_ephys_0
2024-02-13 13:58:10 INFO     training_wheel.py:296  Wheel in rad units using X1 encoding
2024-02-13 13:58:19 WARNING  training_wheel.py:367  no reliable goCue/Feedback times (both needed) for 1 trials
2024-02-13 13:58:20 INFO     training_wheel.py:296  Wheel in rad units using X1 encoding
2024-02-13 13:58:29 WARNING  training_wheel.py:367  no reliable goCue/Feedback times (both needed) for 1 trials
2024-02-13 13:58:29 INFO     bpod_trials.py:102  session extracted 

2024-02-13 13:58:29 WARNING  ephys_fpga.py:134  Keys missing from provided channel map, setting missing keys from default channel map
2024-02-13 13:58:29 WARNING  ephys_fpga.py:777  Attempting to get protocol period from aligning trial start TTLs
2024-02-13 13:58:29 WARNING  ephys_fpga.py:336  Soundcard signal on FPGA seems to have been mixed with 150Hz camera
2024-02-13 13:58:29 INFO     ephys_fpga.py:1188 Attempting to align Bpod clock to DAQ using trial event "intervals_0"
2024-02-13 13:58:29 INFO     ephys_fpga.py:1208 N trials: 1213 bpod, 1214 FPGA, 1213 merged, sync 1.97719 ppm
2024-02-13 13:58:30 INFO     training_wheel.py:296  Wheel in rad units using X4 encoding
2024-02-13 13:58:45 INFO     task_metrics.py:179  Session /Users/gaelle/Downloads/ONE/alyx.internationalbrainlab.org/zadorlab/Subjects/CSH_ZAD_029/2020-09-19/001: Running QC on behavior data...
```

```
The following checks were labelled WARNING:
errorCue_delays
goCue_delays 

The following checks were labelled NOT_SET:
iti_delays
passed_trial_checks 

The following *critical* checks did not pass: [None]
```

The image below is from trial 8, where the error cue is visible. 

![Image](https://github.com/int-brain-lab/paper-brain-wide-map/assets/43007596/bda291f2-29da-46d3-b5c2-4278224c07d6)

I can see the error cue and the red line (indicating the times) fine for a couple trials following, as well for a random late trial 899.

The extraction run upon task QC viewer launching seems fine.
Can the trial extraction task be re-started on Alyx?

**Comment by GaelleChapuis** [comment]
For session EID : a92c4b1d-46bd-457e-a1f4-414265f0e2d4

The response seems to occur always after the go cue, which is what is expected.
How is the reaction time computed ?

One thing is odd though on several trials for this session: the frame2ttl does not flip when the wheel is moving, hence the stimuli freeze is assigned much earlier on.
This is trial 5 as an example.

![Image](https://github.com/int-brain-lab/paper-brain-wide-map/assets/43007596/a7e9fa98-3ab8-4582-a229-673e16fc92f7)
